src/lib/database-migrations/008-spider-data-catalog-errors.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE spider_data_catalog_error (
+    url TEXT NOT NULL,
+    error TEXT NOT NULL,
+    error_at TIMESTAMP WITHOUT TIME ZONE NOT NULL default (now() at time zone 'utc'),
+    found_via JSON NOT NULL
```

Comment:
Wouldn't we want a fk here to the publisher?

Reply:
Because it might be an error at the level of getting a data catalogue, and at that point there is no publisher. Also even an error when parsing a publisher would result in no publisher being written.